File: bubblesort.py
import random

blocks = []
for i in range(1,11):
    num = random.randint(0,6)
    blocks.append(num)

print(blocks)

def bubblesort(array):
    for i in range(0,10):
        for j in range(0,10):
            if array[i] < array[j]:
                array[i], array[j] = array[j], array[i]


bubblesort(blocks)
print(blocks)  

File: bubblesort.js
let blocks = [];
for (let i = 0; i < 11; i++) {
    let number = Math.floor(Math.random(0,6) * 10);
    blocks[i] = number;
}
console.log(blocks);

function bubblesort(array) {
    for (let i = 0; i < array.length; i++) {
        for (let j = 0; j < array.length; j++) {
            if (array[j] > array[j+1]) {
                let temp = array[j];
                array[j] = array[j+1];
                array[j+1] = temp;
            }
        }
    }
}


bubblesort(blocks);
console.log(blocks);
